arci-ros2/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "arci-ros2"
+version = "0.0.5"
+authors = ["Takashi Ogura <ogura@smilerobotics.com>"]
+edition = "2018"
+license = "Apache-2.0"
+description = "arci implementation using ROS2"
+keywords = ["robotics", "robot"]
+categories = ["science::robotics"]
+repository = "https://github.com/openrr/openrr"
+documentation = "http://docs.rs/arci-ros2"
+
+[dependencies]
+anyhow = "1.0"
+arci = "0.0.5"
+async-trait = "0.1"
+thiserror = "1.0"
+r2r = { git = "https://github.com/sequenceplanner/r2r", optional = true}
+
+[dev-dependencies]
+tokio = { version = "1.0", features = ["full"] }
```

Comment:
@taiki-e Is it OK to use full?

Reply:
Yes, it's fine. We can adjust it later if necessary. (Also, I guess the relevant C++ code or bindings probably won't work with WASM.)